src/meltano/cli/cli.py

```diff
@@ -123,3 +118,38 @@ def cli(  # noqa: WPS231
             "For more details, visit https://docs.meltano.com/guide/installation#upgrading-meltano-version"
         )
         sys.exit(3)
+
+
+def activate_environment(ctx: click.Context, project: Project) -> None:
+    """Activate the selected environment.
```

Comment:
@WillDaSilva - What do you think of slightly expanding this docstring to make it clear default environments are also applied? Maybe:

```suggestion
    """Activate the requested environment or a default environment if nothing requested.
```

Reply:
Thanks for the suggestion. I updated it in b46ebc6 with the elaboration on the next line, because I couldn't fit a satisfactory summary all in the first line.